python/graphstorm/model/gnn.py

```diff
@@ -741,20 +743,27 @@ def unfreeze_input_encoder(self):
             self._edge_input_encoder.unfreeze()
 
     # pylint: disable=signature-differs
-    def restore_dense_model(self, restore_model_path,
-                            model_layer_to_load=None):
-        # TODO(zhengda) we need to load edge_input_encoder.
+    def restore_dense_model(self, restore_model_path, model_layer_to_load=None):
+        """ restore dense models
```

Comment:
What about adding .. versionchanged:: to the log here?

Reply:
Good call. Changed.